nav2_costmap_2d/src/costmap_2d_ros.cpp

```diff
@@ -70,6 +70,7 @@ Costmap2DROS::Costmap2DROS(const std::string & name, tf2_ros::Buffer & tf)
 {
   node_ = std::shared_ptr<rclcpp::Node>(this, [](rclcpp::Node *) {});
 
+#if 0
```

Comment:
Are these costmap_2d_ros changes supposed to be in here? This looks like it's here by mistake.

Reply:
No, these shouldn't be here.  I will remove these from this PR.